operator/config/autopilot-channels/namespaced-main/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resources:
+- ../release-bundle/install-bundle-autopilot-namespaced/0-cnrm-system.yaml
+
+patchesJson6902:
```

Comment:
super nit: `patches` contains `patchesJson6902` and `patchesStrategicMerge`. I saw this PR mixes the usage of `patches` and `patchesJson6902`, may be better to align them two.

Reply:
Yes, I tackled the ones that I thought were critical but there are still some to go.

Once we aren't running kustomize as part of the release itself (I.e. once we are committing the output of kustomize, as we are doing here), then I think we should feel much more comfortable about making bigger changes to our kustomizations.